tests/prague/eip7702_set_code_tx/test_set_code_txs_2.py

```diff
@@ -1750,3 +1750,46 @@ def test_set_code_type_tx_pre_fork(
             ),
         },
     )
+
+
+@pytest.mark.valid_from("Prague")
+def test_delegation_replacement_call_previous_contract(
+    state_test: StateTestFiller,
+    pre: Alloc,
+):
+    """
+    Test setting the code of an EOA that already has
+    delegation, calling the previous delegated contract.
+    Previous contract shouldn't be warm when doing the CALL.
+    """
+    pre_set_delegation_code = Op.RETURN(0, 0)
```

Comment:
```suggestion
    pre_set_delegation_code = Op.STOP
```
I'd change this to only using stop so it consumes zero gas.

Reply:
Changed it [here](https://github.com/ethereum/execution-spec-tests/pull/1648/commits/c1c23006a8e96c4cd570564aafd3e97140b36df1)